### ISSUE TITLE
`mouse update` doesn't update

### ISSUE THREAD

**Issue by Kiedtl** [lptstr/mouse#14]
When updating Mouse with the `mouse update` command, the following output is observed:
```
kiedtl ~ ❯❯❯ mouse update
fatal: not a git repository (or any of the parent directories): .git
Updating Mouse... done
Successfully updated Mouse.
fatal: not a git repository (or any of the parent directories): .git
git-crypt: Error: 'git status' failed - is this a Git repository?
kiedtl ~ ❯❯❯
```
I appears that Mouse is `cd`ing to the wrong location.

**Comment by Kiedtl** [comment]
Look at this [line](https://github.com/Kiedtl/mouse/blob/7188dad47199be4fb900a8e0fa4400d39299e8c4/libexec/mouse-update.ps1#L25) of code. It looks like Mouse is trying to get the Git branch before `cd`ing to the write location.
Also, I thinl that the git-crypt error comes from the entrypint in the finally block, which tries to lock the repository without `cd`ing there.

**Comment by Kiedtl** [comment]
Looks like this bug was fixed in commit 0c95b8fd749626166ec86af10210bd83818a833a.
